Documentation/11/update gh pages

helen-d-little opened the pull request:
Reorganised some documents, not complete.
Needs to be pushed to main before visible on ghpages
Nothing that should be NATS sensitive.

Commit:
build for new branch added to yml file

Commit:
update intro

Commit:
Update intro add images

Commit:
update images

Commit:
formatting changes and rearrangement

Commit:
rearrangement some duplication removed

radka-j commented on intro/about.md:
```diff
@@ -0,0 +1,20 @@
+## Aviary
+[![Build Status](https://travis-ci.com/project-bluebird/aviary.svg?branch=develop)](https://travis-ci.com/alan-turing-institute/aviary)
+
+[Aviary](../reference/aviary.md) is an air traffic scenario generation package developed as part of the BlueBird project. It aims to make available a large number of simple (but non-trivial) air traffic control scenarios suitable for the training of an artificial agent via reinforcement learning. The package also includes a set of evaluation metrics for measuring performance in the air traffic co ...
```
I'd remove reinforcement learning from this section - we provide platform for agent training, it's up to people what specific method they want to use

helen-d-little replied:
Yup updated text and will put on next push (this afternoon)

radka-j commented on intro/install.md:
```diff
@@ -1,3 +1,154 @@
+# Installation Instructions
+With an air traffic control (ATC) simulator (e.g., BlueSky) and BlueBird running, one can observe and interact with the simulation via BlueBird using Python (PyDodo), R (rdodo) or Twitcher. The Aviary package allows one to synthetically generate ATC scenarios of increasing complexity to train agents on and provides metrics to score performance.
+
+## Quick Start Using Simurgh
```
Should we (can we) move the example files to this repo and just get rid of simurgh entirely? The point of simurgh was to be the over-arching docs repo and the only thing it contains are the docker files and the example notebook. Ideally those would be all here and we can get rid of it. That said, I understand that this repo launches GitHub page, so maybe we don't want to ask people to have to clone this repo and *do* want a link to some other repo to clone. In that case, maybe we just rename and clean up simurgh. Which do you think is better?

helen-d-little replied:
Yes I agree - I think there is a lot of overlap between what simurgh does and what the ghpages will do.
I think copying over the contents of simaugh would make sense, as long as there is somewhere sensible for the docker files to go. So I'm not sure what the best option would be. I think the docs repo is public as it has to be in order for the pages to be launched so there's no reason that I can think of to not have them here, but there could be good reason to leave them be?

radka-j commented on reference/aviary.md:
```diff
@@ -2,22 +2,23 @@
 [![Build Status](https://travis-ci.com/project-bluebird/aviary.svg?branch=develop)](https://travis-ci.com/alan-turing-institute/aviary)
 
-Aviary is an air traffic scenario generation package developed as part of the [Simurgh](https://github.com/alan-turing-institute/simurgh) project. It aims to make available a large number of simple (but non-trivial) air traffic control scenarios suitable for the training of an artificial agent via reinforcement learning. The package also includes a set of evaluation metrics for measuring performa ...
+Aviary is an air traffic scenario generation package developed as part of the BlueBird project. It aims to make available a large number of simple (but non-trivial) air traffic control scenarios suitable for the training of an artificial agent via reinforcement learning. The package also includes a set of evaluation metrics for measuring performance in the air traffic control challenge.
```
again, i'd remove mention of RL here

helen-d-little replied:
Oddly enough I had changed it to BlueBird but obviously not in this push! RL removed based on above

radka-j reviewed (commented):
Looks good!! :)

Commit:
Edit to docs after Radka Review

Removal of comments about reinforcement learning with aviary.
Simurgh location and documentation still undecided
